script/Arcueid/Day 10/10_00_ARC10_4.txt

```diff
@@ -752,24 +753,23 @@ But you said that you hate sucking blood, right?
 -- Page 348, Offset 25138.
 -- あ。そうか、人間に食べ物の好き嫌いがあるように、アルクェイドは血の味が嫌いってワケだな！？」
 Ah, I got it!
- All humans have their own individual likes and dislikes when it comes to food―――
+ All humans have their preferences when it comes to food―――
 Could it be that you just hate the taste of blood?"
 }
 [sha:9df9f0612940ed4bed52907549314efd01067782]{
 -- Page 349, Offset 25139.
 -- 会心の閃きを口にする。
-I blurt out my satisfying revelation.
+I come to a revelation that feels convicing and blurt it out.
 }
 [sha:019643faeef74e7ec3ccb1725d3b703bca8cee2b]{
 -- Page 349, Offset 25140.
 -- しかし、アルクェイドは辛そうに眉をひそめた。
-// Kinda hate 辛い
 However, Arcueid frowns as though something is weighing on her.
 }
 [sha:70840069c591988e9f870b8b707a9db6eeb858a9]{
 -- Page 350, Offset 25141. Mods: @n.
 -- 「……わからない。血の味なんて、
-"...I don't know. The taste of blood―――
+"I don't know... the taste of blood―――
```

Comment:
Recapitalise 'the'. It currently reads like 'I don't know... the taste of blood. ' is the sentence, when it should be split up like 'I don't know... The taste of blood is unknown to me.'

Reply:
I think that's splitting hairs? current ver. is fine